.github/workflows/release.yml

```diff
@@ -41,6 +40,56 @@ jobs:
           python setup.py sdist bdist_wheel
           twine upload dist/*
 
+  publish-merlin-pyfunc-server:
+    # if: ${{ startsWith(github.ref, 'refs/tags/') }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.8
+          cache: pip
+      - name: Install dependencies
+        working-directory: ./python/pyfunc-server
+        run: pip install setuptools>=64 setuptools_scm>=8 twine wheel
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.pypi_username }}
+          TWINE_PASSWORD: ${{ secrets.pypi_password }}
+        working-directory: ./python/pyfunc-server
+        run: |
+          tag=$(python -m setuptools_scm -r ../.. | sed 's/\+.*//')
+          sed -i -e "s|VERSION = \".*\"|VERSION = \"`echo "${tag}"`\"|g" ./pyfuncserver/version.py
+          python setup.py sdist bdist_wheel
+          twine upload dist/*
```

Comment:
thinking publishing to pypi every ci will be too much

Reply:
is it possible to release on tag only? like our sdk

Reply:
Yes, that's the goal. I've commented this condition to only run the jobs on tag: https://github.com/caraml-dev/merlin/pull/547/files/00277869339e33359b24e8a2f4b8cbe70691d32b#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R44

Will put TODO there as well.